fisher.lua

```diff
@@ -421,16 +428,31 @@ function check_incoming_chunk(id, original, modified, injected, blocked)
             if current.monster == false and fish:with('bite_id', current.bite_id) then
                 current.key = original:sub(21)
                 stats.bites = stats.bites + 1
-                local delay = fish:with('bite_id', current.bite_id).delay
```

Comment:
Short circuit logic could be used here again instead of needed the if/else block.